openedx/core/djangoapps/schedules/migrations/0006_scheduleexperience.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('schedules', '0005_auto_20171010_1722'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ScheduleExperience',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('experience_type', models.PositiveSmallIntegerField(default=0, choices=[(0, b'Recurring Nudge and Upgrade Reminder'), (1, b'Course Updates')])),
```

Comment:
Question: We expect to add to the list of experience_type choices over time.  Changes to the `choices` value shouldn't cause a full database migration of the entire `ScheduleExperience` table, right?

Reply:
Pinging @ormsbee in case he knows offhand.

Reply:
My understanding is that the database doesn't know about choices at all, so changing them should not result in a SQL migration being generated. I'll test this on my devstack to be sure.

Reply:
OK, I added a third option, ran "makemigrations" and got this:

```python
# -*- coding: utf-8 -*-
from __future__ import unicode_literals

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('schedules', '0006_scheduleexperience'),
    ]

    operations = [
        migrations.AlterField(
            model_name='scheduleexperience',
            name='experience_type',
            field=models.PositiveSmallIntegerField(default=0, choices=[(0, b'Recurring Nudge and Upgrade Reminder'), (1, b'Course Updates'), (2, b'Another experience')]),
        ),
    ]
```

I then ran `./manage.py lms --settings devstack_docker sqlmigrate schedules 0007` and it produced no output, which I think means that no changes would actually be made to the database.

Either @ormsbee or @nasthagiri - is this a valid check?

Also note that I ran sqlmigrate on migration 6 (which is committed in this PR) and got this:

```sql
BEGIN;
CREATE TABLE `schedules_scheduleexperience` (`id` integer AUTO_INCREMENT NOT NULL PRIMARY KEY, `experience_type` smallint UNSIGNED NOT NULL, `schedule_id` integer NOT NULL UNIQUE);
ALTER TABLE `schedules_scheduleexperience` ADD CONSTRAINT `schedules__schedule_id_5ca03607383f8535_fk_schedules_schedule_id` FOREIGN KEY (`schedule_id`) REFERENCES `schedules_schedule` (`id`);

COMMIT;
```

It has no references to the actual choice values.